package.json

```diff
@@ -29,8 +29,8 @@
 		"gulp-coveralls": "0.1.4",
 		"http-server": "^0.11.1",
 		"jest": "26.4.2",
-		"liferay-frontend-theme-styled": "5.0.0",
-		"liferay-frontend-theme-unstyled": "5.0.0",
+		"liferay-frontend-theme-styled": "6.0.1-test.1c962d5e12be",
+		"liferay-frontend-theme-unstyled": "6.0.1-test.1c962d5e12be",
```

Comment:
Before we go final with this, do we want to publish non-`test` releases? Or do we have to wait for 7.4 release to do that? (ie. because we want to make sure that `@latest` continues to point at the v5 versions?)

Reply:
From my understanding, we shouldn't release anything under the `latest` tag manually. I was under the impression that we wait for 7.4 to release those.

It is a little odd to be testing off of a `beta` release, but since we are the only ones using these tests(infra team) I don't think it would be a major issue.